packages/sdk-js/src/handlers/auth.ts

```diff
@@ -83,6 +87,9 @@ export class AuthHandler {
 	async logout() {
 		await this.axios.post('/auth/logout');
 		this.token = null;
+		if (this.mode === 'json'){
```

Comment:
This is a good idea. Can you add it to my PR? Only problem is that there is no `storage.removeItem` method, maybe we can simply `setItem('directus_refresh_token', null)`?

Reply:
you do it better..  I take a step back.